src/types.d.ts

```diff
@@ -808,19 +827,27 @@ declare namespace bigquery {
      * [Output-only] A URL that can be used to access the resource again. You can use this URL in Get or Update requests to the resource.
      */
     selfLink?: string;
+    /**
```

Comment:
Are these generated? It might be nice to break out the {} here if not.

Reply:
Yeah, they're generated.